Assets/Scripts/Units/Javelin Unit/Javelin.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RogueApeStudio
+{
+    public class Javelin : MonoBehaviour
+    {
+        [SerializeField] private Rigidbody _rb;
+        [SerializeField] private Collision _target;
+        [SerializeField] private float _despawnDuration = 1f;
+        [SerializeField] private float _playerLayer = 9;
```

Comment:
This hardcoded value could change in the future, might want to search for a method that grabs the value.

Reply:
fixed